broadinstitute/pearl
# JN-321 fixing participant view link

The link from the admin tool to the participant view was broken.  To fix it, this adds the deployment zone hostname variables to the config object passed to the frontend.  This also cleans up various other broken links in the admin tool that resulted from porting them over from the prototype.

TO TEST:
1. restart adminApiApp
2. go to https://localhost:3000/ourhealth/studies/ourheart/env/sandbox
3. click 'participant view' at the top of the page
4. confirm you are taken to th participant landing page
5. back in the admin tool, open the side bar by clicking the hamburger in the top left.  confirm the links work.   (Note that Erin and I are in agreement that we should find a better UI paradigm than the burger+sidebar for future admin tool work, but this is what we've got for now)
